packages/components/psammead-brand/src/index.jsx

```diff
@@ -145,32 +148,21 @@ StyledBrand.defaultProps = {
   serviceLocalisedName: null,
 };
 
-const Brand = ({
-  product,
-  serviceLocalisedName,
-  svgHeight,
-  minWidth,
-  maxWidth,
-  svg,
-  url,
-}) => {
-  const styledBrandProps = {
-    product,
-    serviceLocalisedName,
-    svgHeight,
-    minWidth,
-    maxWidth,
-    svg,
-  };
+const Brand = props => {
+  const { svgHeight, maxWidth, minWidth, url, borderTop, borderBottom } = props;
```

Comment:
This is nice! I think this looks a lot better than unpacking and repacking `props` :+1: